guides/source/upgrading_ruby_on_rails.md

```diff
@@ -66,6 +66,17 @@ Overwrite /myapp/config/application.rb? (enter "h" for help) [Ynaqdh]
 
 Don't forget to review the difference, to see if there were any unexpected changes.
 
+Upgrading from Rails 5.2 to Rails 6.0
+-------------------------------------
+
+### Force SSL
+
+The `force_ssl` method on controllers has been deprecated and will be removed in
+Rails 6.0. you are encouraged to enable `config.force_ssl` to enforce HTTPS
```

Comment:
capital 'Y'

Unrelated: I enjoy your show 🚲 🔔 and I am looking forward to your talk at RailsConf.

Reply:
Another place here re: 6.0 -> 6.1.

Reply:
One more thing — Look like "You" in "you are encouraged ..." has to be capitalized.